File: Php/app/Cashgame/Matrix/ColumnHeaderModel.php
namespace app\Cashgame\Matrix{

	use entities\Homegame;
	use app\Urls\CashgameDetailsUrlModel;
	use core\Globalization;
	use entities\Cashgame;

	class ColumnHeaderModel{

		public $date;
		public $cashgameUrl;

		public function __construct(Homegame $homegame, Cashgame $cashgame, $showYear = false){
			date = Globalization::formatShortDate($cashgame.getStartTime(), $showYear);
			cashgameUrl = new CashgameDetailsUrlModel($homegame, $cashgame);
		}

	}

}

File: Web/Models/ColumnHeaderModel.cs
using Core.Classes;
using Infrastructure.System;
using Web.Models.Url;
using app;

namespace Web.Models{
    public class ColumnHeaderModel{

	    public string Date { get; set; }
	    public UrlModel CashgameUrl { get; set; }

		public ColumnHeaderModel(Homegame homegame, Cashgame cashgame, bool showYear = false){
            Date = cashgame.StartTime.HasValue ? Globalization.FormatShortDate(cashgame.StartTime.Value, showYear) : string.Empty;
			CashgameUrl = new CashgameDetailsUrlModel(homegame, cashgame);
		}

	}

}